.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Remove this file from, we dont want ide configs in the git repo, it is bad practice